<!DOCTYPE html>
<html lang="en">

<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0" />
    <title>UnJin Lee</title>

    <!-- CSS  -->
    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
    <link href="css/materialize.css" type="text/css" rel="stylesheet" media="screen,projection" />
    <link href="css/style.css" type="text/css" rel="stylesheet" media="screen,projection" />
</head>

<body>
    <nav class="maroon" role="navigation">
        <div class="nav-wrapper container"><a id="logo-container" href="#" class="brand-logo">UnJin Lee</a>
            <ul class="right hide-on-med-and-down">
                <li><a href="#education">Education</a></li>
                <li><a href="#pubs">Publications</a></li>
                <li><a href="#extra">Achievements</a></li>
                <li><a href="#aboutme">About Me</a></li>
            </ul>

            <ul id="nav-mobile" class="side-nav">
                <li><a href="#education" class="mobile-link">Education</a></li>
                <li><a href="#pubs" class="mobile-link">Publications</a></li>
                <li><a href="#extra" class="mobile-link">Achievements</a></li>
                <li><a href="#aboutme" class="mobile-link">About Me</a></li>
            </ul>
            <a href="#" data-activates="nav-mobile" class="button-collapse"><i class="material-icons">menu</i></a>
        </div>
    </nav>
    <div class="section no-pad-bot" id="index-banner">
        <div class="container1">
            <div class="cover-img">
                <div class="mug">
                    <img class="responsive-img circle" src="https://i.imgur.com/CxuupC1.jpg">
                </div>
            </div>
            <div class="section" id="education">
                <h1 class="header center maroon-text">UnJin Lee</h1>
                <h4 class="center">Postdoctoral Associate, <a class="underline black-text" href="https://zhaolab.rockefeller.edu/index.php/people/" target="_blank">Laboratory of Evolutionary Genetics and Genomics</a></h4>
                <h4 class="center">Rockefeller University</h4>
                <h4 class="center grey-text">PhD, MA University of Chicago, 2021</h4>
                <h5 class="center grey-text">Committee on Genetics, Genomics, and Systems Biology</h5>
            </div>
        </div>
    </div>

    <div class="container">
        <div class="section" id="pubs">
            <!--   Publication Section   -->
            <h3 class="center maroon-text">Publications</h3>
            <div class="row">
                <div class="col s12 m6">
                    <div class="card hoverable">
                        <div class="card-image waves-effect waves-block waves-light">
                            <img class="activator" src="https://ars.els-cdn.com/content/image/1-s2.0-S0303264722001721-gr4.jpg" />
                        </div>
                        <div class="card-content">
                            <span class="card-title activator grey-text text-darken-4">Evolution and maintenance of phenotypic plasticity<i class="material-icons right">more_vert</i></span>
                        </div>
                        <div class="card-action">
                            <p><a class="btn waves-effect waves-light dark-grey" target="_blank" href="https://www.sciencedirect.com/science/article/pii/S0303264722001721">ScienceDirect</a></p>
                        </div>
                        <div class="card-reveal">
                            <span class="card-title grey-text text-darken-4">Evolution and maintenance of phenotypic plasticity<i class="material-icons right">close</i></span>
                            <p>We introduce a novel framework for exploring the evolutionary consequences of phenotypic plasticity (adaptive and non-adaptive) integrating both genic and epigenetic effects on phenotype via stochastic differential equations
                                and in-silico selection.</p>
                        </div>
                    </div>
                </div>
                <div class="col s12 m6">
                    <div class="card hoverable">
                        <div class="card-image waves-effect waves-block waves-light">
                            <img class="activator" src="https://www.researchgate.net/profile/Jian-Zu-2/publication/332048832/figure/fig1/AS:750177577103361@1555867680074/A-part-of-the-gene-age-specific-coexpression-network-in-human-Each-circle-corresponds-to.ppm" />
                        </div>
                        <div class="card-content">
                            <span class="card-title activator grey-text text-darken-4">Topological evolution of coexpression networks by new gene integration maintains the hierarchical and modular structures in human ancestors <i class="material-icons right">more_vert</i></span>
                        </div>
                        <div class="card-action">
                            <p><a class="btn waves-effect waves-light dark-grey" target="_blank" href="https://www.researchgate.net/publication/332048832_Topological_evolution_of_coexpression_networks_by_new_gene_integration_maintains_the_hierarchical_and_modular_structures_in_human_ancestors">Sci China Life Sci</a></p>
                        </div>
                        <div class="card-reveal">
                            <span class="card-title grey-text text-darken-4">Topological evolution of coexpression networks by new gene integration maintains the hierarchical and modular structures in human ancestors<i class="material-icons right">close</i></span>
                            <p>We analyze the global structure and evolution of human gene coexpression networks driven by new gene integration. When the Pearson correlation coefficient is greater than or equal to 0.5, we find that the coexpression network
                                consists of 334 small components and one "giant" connected subnet comprising of 6317 interacting genes. This network shows the properties of power-law degree distribution and small-world. The average clustering coefficient
                                of younger genes is larger than that of the elderly genes (0.6685 vs. 0.5762). Particularly, we find that the younger genes with a larger degree also show a property of hierarchical architecture. The younger genes play
                                an important role in the overall pivotability of the network and this network contains few redundant duplicate genes. Moreover, we find that gene duplication and orphan genes are two dominant evolutionary forces in shaping
                                this network. Both the duplicate genes and orphan genes develop new links through a "rich-gets-richer" mechanism. With the gradual integration of new genes into the ancestral network, most of the topological structure features
                                of the network would gradually increase. However, the exponent of degree distribution and modularity coefficient of the whole network do not change significantly, which implies that the evolution of coexpression networks
                                maintains the hierarchical and modular structures in human ancestors. </p>
                        </div>
                    </div>
                </div>

            </div>
        </div>

        <div class="row">
            <div class="col s12 m6">
                <div class="card hoverable">
                    <div class="card-image waves-effect waves-block waves-light">
                        <img class="activator" src="https://journals.plos.org/plosgenetics/article/figure/image?size=large&id=10.1371/journal.pgen.1009654.g001" />
                    </div>
                    <div class="card-content">
                        <span class="card-title activator grey-text text-darken-4">Genomic analyses of new genes and their phenotypic effects reveal rapid evolution of essential functions in Drosophila development<i class="material-icons right">more_vert</i></span>
                    </div>
                    <div class="card-action">
                        <p><a class="btn waves-effect waves-light dark-grey" target="_blank" href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009654">Plos Genetics</a></p>
                    </div>
                    <div class="card-reveal">
                        <span class="card-title grey-text text-darken-4">Genomic analyses of new genes and their phenotypic effects reveal rapid evolution of essential functions in Drosophila development<i class="material-icons right">close</i></span>
                        <p>It is a conventionally held dogma that the genetic basis underlying development is conserved in a long evolutionary time scale. Ample experiments based on mutational, biochemical, functional, and complementary knockdown/knockout
                            approaches have revealed the unexpectedly important role of recently evolved new genes in the development of Drosophila. The recent progress in the genome-wide experimental testing of gene effects and improvements in the computational
                            identification of new genes (
                            < 40 million years ago, Mya) open the door to investigate the evolution of gene essentiality with a phylogenetically high resolution</p>
                    </div>
                </div>
            </div>

            <div class="col s12 m6">
                <div class="card hoverable">
                    <div class="card-image waves-effect waves-block waves-light">
                        <img class="activator" src="https://i.imgur.com/E9vKJOc.png">
                    </div>
                    <div class="card-content">
                        <span class="card-title activator grey-text text-darken-4"> Prognostic and predictive breast cancer signature <i class="material-icons right">more_vert</i></span>
                    </div>
                    <div class="card-action">
                        <p><a class="btn waves-effect waves-light dark-grey" target="_blank" href="https://patents.google.com/patent/US20160078167A1/en">USPTO</a></p>
                    </div>
                    <div class="card-reveal">
                        <span class="card-title grey-text text-darken-4"> Prognostic and predictive breast cancer signature <i class="material-icons right">close</i></span>
                        <p>Embodiments of the invention are directed to methods of determining the prognosis of a breast cancer patient by evaluating a specified set of genes. Specifically, methods may comprise calculating a prognosis score based on a particular
                            algorithm. Also disclosed are compositions, kits and methods for treating cancer in a subject in need thereof are disclosed involving one or more upstream activators and/or downstream effectors of TET1.</p>
                    </div>
                </div>
            </div>

        </div>

        <div class="row">


            <div class="col s12 m6">
                <div class="card hoverable">
                    <div class="card-image waves-effect waves-block waves-light">
                        <img class="activator" src="https://i.imgur.com/MLn2Cie.png">
                    </div>
                    <div class="card-content">
                        <span class="card-title activator grey-text text-darken-4">A The Cyanobacterial Circadian Clock Follows Midday in Vivo and in Vitro<i class="material-icons right">more_vert</i></span>
                    </div>
                    <div class="card-action">
                        <p><a class="btn waves-effect waves-light dark-grey" target="_blank" href="https://elifesciences.org/articles/23539">eLIFE</a></p>
                    </div>
                    <div class="card-reveal">
                        <span class="card-title grey-text text-darken-4">The Cyanobacterial Circadian Clock Follows Midday in Vivo and in Vitro<i class="material-icons right">close</i></span>
                        <p>Here we report experimental platforms for driving the cyanobacterial circadian clock both in vivo and in vitro. We find that the phase of the circadian rhythm follows a simple scaling law in light-dark cycles, tracking midday across
                            conditions with variable day length. The core biochemical oscillator comprised of the Kai proteins behaves similarly when driven by metabolic pulses in vitro, indicating that such dynamics are intrinsic to these proteins. We
                            develop a general mathematical framework based on instantaneous transformation of the clock cycle by external cues, and it successfully predicts clock behavior under many cycling environments.</p>
                    </div>
                </div>
            </div>

            <div class="col s12 m6">
                <div class="card hoverable">
                    <div class="card-image waves-effect waves-block waves-light">
                        <img class="activator" src="http://journals.aps.org/pre/article/10.1103/PhysRevE.93.062127/figures/4/medium">
                    </div>
                    <div class="card-content">
                        <span class="card-title activator grey-text text-darken-4">Geometric Structure and Geodesic in a Solvable Model of Nonequilibrium Process<i class="material-icons right">more_vert</i></span>
                    </div>
                    <div class="card-action">
                        <p><a href="http://journals.aps.org/pre/abstract/10.1103/PhysRevE.93.062127" class="btn waves-effect waves-light dark-grey" target="_blank">Phys Rev E</a></p>
                    </div>
                    <div class="card-reveal">
                        <span class="card-title grey-text text-darken-4">Geometric Structure and Geodesic in a Solvable Model of Nonequilibrium Process<i class="material-icons right">close</i></span>
                        <p>We investigate the geometric structure of a nonequilibrium process and its geodesic solutions. By employing an exactly solvable model of a driven dissipative system (generalized nonautonomous Ornstein-Uhlenbeck process), we compute
                            the time-dependent probability density functions (PDFs) and investigate the evolution of this system in a statistical metric space where the distance between two points (the so-called information length) quantifies the change
                            in information along a trajectory of the PDFs. </p>
                    </div>
                </div>
            </div>
        </div>

        <div class="row">
            <div class="col s12 m6">
                <div class="card hoverable">
                    <div class="card-image waves-effect waves-block waves-light">
                        <img class="activator" src="http://i.imgur.com/fg4c6dB.png">
                    </div>
                    <div class="card-content">
                        <span class="card-title activator grey-text text-darken-4">Noise-Driven Phenotypic Heterogeneity with Finite Correlation Time in Clonal Populations<i class="material-icons right">more_vert</i></span>
                    </div>
                    <div class="card-action">
                        <p><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0132397" class="btn waves-effect waves-light dark-grey" target="_blank">plos one</a></p>
                    </div>
                    <div class="card-reveal">
                        <span class="card-title grey-text text-darken-4">Noise-Driven Phenotypic Heterogeneity with Finite Correlation Time in Clonal Populations<i class="material-icons right">close</i></span>
                        <p>There has been increasing awareness in the wider biological community of the role of clonal phenotypic heterogeneity in playing key roles in phenomena such as cellular bet-hedging and decision making, as in the case of the phage-λ
                            lysis/lysogeny and B. Subtilis competence/vegetative pathways. Here, we report on the effect of stochasticity in growth rate, cellular memory/intermittency, and its relation to phenotypic heterogeneity.</p>
                    </div>
                </div>
            </div>
            <div class="col s12 m6">
                <div class="card hoverable">
                    <div class="card-image waves-effect waves-block waves-light">
                        <img class="activator" src="http://i.imgur.com/wdRCWcA.png">
                    </div>
                    <div class="card-content">
                        <span class="card-title activator grey-text text-darken-4">A Prognostic Gene Signature for Metastasis-Free Survival of Triple Negative Breast Cancer Patients<i class="material-icons right">more_vert</i></span>
                    </div>
                    <div class="card-action">
                        <p><a class="btn waves-effect waves-light dark-grey" target="_blank" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0082125">PLOS ONE</a></p>
                    </div>
                    <div class="card-reveal">
                        <span class="card-title grey-text text-darken-4">A Prognostic Gene Signature for Metastasis-Free Survival of Triple Negative Breast Cancer Patients<i class="material-icons right">close</i></span>
                        <p>The application of gene expression array technology to breast cancer has emphasized the heterogeneity of this disease and also provided new tools to classify breast cancers into subtypes based on gene expression patterns. Ideally
                            each subtype would reflect distinct molecular characteristics corresponding to discrete cancer phenotypes. This information could be used to gain prognostic insight and, eventually, to predict response to therapy.</p>
                    </div>
                </div>
            </div>
        </div>

        <!--Extra-->
        <div class="section" id="extra">
            <h3 class="center maroon-text">Achievements</h3>
            <div class="row">
                <div class="col s12 m6 extra">
                    <h5 class="maroon-text center">Awards & Honors</h5>
                    <ul>
                        <li class="center">NIH T32 Gene Regulation Training Grant Recipient</li>
                        <li class="center">NSF GRFP Honorable Mention 2016</li>
                        <li class="center">Odyssey Scholar</li>
                        <li class="center">National Merit Scholar</li>
                    </ul>
                </div>
                <div class="col m6 s12 extra">
                    <h5 class="maroon-text center">Academic Service Activities</h5>
                    <ul>
                        <li class="center">Co-Host Groks Science Show Podcast</li>
                        <li class="center">Co-Host Groks Science Show Radio Hour WHPK 88.5FM</li>
                        <li class="center">Genetics Science Connections at the Museum of Science and Industry</li>
                        <li class="center">American Association for the Advancement of Science Member</li>
                    </ul>
                </div>
                <div class="col m6 s12 extra">
                    <h5 class="maroon-text center">Experimental Techniques, Expertise</h5>
                    <ul>
                        <li class="center">Stochastic Differential Equations</li>
                        <li class="center">Applied Numerical Optimization</li>
                        <li class="center">Monte Carlo Methods</li>
                        <li class="center">Gene Expression Analysis</li>
                        <li class="center">Survival Analysis</li>
                        <li class="center">Statistical Modeling</li>
                        <li class="center">Chromosomal Confirmation Capture (4C)</li>
                        <li class="center">Enhancer-Reporter Assays</li>
                        <li class="center">Phylogenetic Analysis</li>
                        <li class="center">Cloning</li>
                    </ul>
                </div>
                <div class="col m6 s12 extra">
                    <h5 class="maroon-text center">Presentations</h5>
                    <ul>
                        <li class="center">2013 Undergraduate Symposium of the American Association of Cancer Researchers (Poster)</li>
                        <li class="center">2013 University of Chicago (Poster) Undergraduate Research Symposium</li>
                        <li class="center">2016 Aspen Center for Physics Winter Conference on Evolution, Populations, and Physics (Poster)</li>
                    </ul>
                </div>
            </div>
            <div class="row">
                <div class="col s12 m12 extra">
                    <h5 class="maroon-text center">Software</h5>
                    <h6 class="maroon-text center">Scientific Software</h6>
                    <ul>
                        <li class="center"><a class="maroon-text" href="https://www.bioconductor.org/packages/release/bioc/html/sigsquared.html">sigsquared</a> (c.f. Lee and Frankenberger et al 2013/R/S4)</li>
                        <li class="center">Deterministic Runge Kutta Solvers (1st-4th order/C++)</li>
                        <li class="center">Stochastic Runge Kutta Solvers (Honeycutt with Finite Correlation Time/C++)</li>
                        <li class="center">Gillespie Stochastic Simulation Algorithm (First Reaction method/C++)</li>
                        <li class="center">Genetic Algorithm (C++)</li>
                        <li class="center">Downhill Simplex using LASSO (Java)</li>
                    </ul>
                    <h6 class="maroon-text center">Languages</h6>
                    <ul id="languages">
                        <li class="center">R (S3/S4)</li>
                        <li class="center">MATLAB</li>
                        <li class="center">C++</li>
                        <li class="center">Java</li>
                        <li class="center">bash</li>
                    </ul>
                </div>
            </div>
        </div>
    </div>
    <div class="section" id="aboutme">
        <footer class="page-footer">
            <div class="container">
                <div class="row">
                    <div class="col s12">
                        <h5 class="white-text center">Bio</h5>
                        <p class="grey-text text-lighten-4 center">
                            UnJin Lee is an evolutionary geneticist with a background in physics. His prior work has been primarily focused on the interface between cancer diagnostics and systems biology, focusing on statistical and numerical methods. As a graduate student in the
                            Committee on Genetics, Genomics, and Systems Biology at the University of Chicago, studied the functional evolution of new, essential genes and the evolution of phenotypic plasticity. Currently he is at the Laboratory of Evolutionary
                            Genetics and Genomics at the Rockefeller University, working on understanding the selective forces that drive non-coding regions of the genome to evolve into new de novo genes.
                        </p>
                    </div>
                </div>
                <div class="row">
                    <div class="col l6 s12">
                        <h5 class="white-text center">Interests</h5>
                        <ul class="center">
                            <li class="white-text">Functional Evolution</li>
                            <li class="white-text">New Gene Origination</li>
                            <li class="white-text">Evolutionary Aspects of Phenotypic Heterogeneity</li>
                        </ul>
                    </div>
                    <div class="col l6 s12">
                        <h5 class="white-text center">Contact</h5>
                        <ul class="center">
                            <i class="contact small light-grey-text material-icons"><a href="mailto:linuxlife@gmail.com" target="blank">email</a></i>
                        </ul>
                    </div>
                </div>
            </div>
            <div class="footer-copyright">
                <div class="container center light-grey-text">
                    created by <a class="sanj-link" href="http://ssatyal.github.io" target="_blank">Sanjaya Satyal</a>
                </div>
            </div>
        </footer>
    </div>
    <!--  Scripts-->
    <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
    <script src="js/materialize.js"></script>
    <script src="js/init.js"></script>
    <script src="js/script.js"></script>

</body>

</html>